packages/ihe-gateway-sdk/src/models/shared.ts

```diff
@@ -30,7 +30,7 @@ export const SamlAttributesSchema = z.object({
 
 export const baseRequestSchema = z.object({
   id: z.string(),
-  cxId: z.string().optional(),
+  cxId: z.string(),
```

Comment:
reverting this back to mandatory and modifying the incoming request types to omit cxId